gateway/helm/values.yaml

```diff
@@ -92,22 +92,22 @@ ingress:
 # https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
 resources:
   requests:
-    cpu: 1
-    memory: 256Mi
+    cpu: 50m
+    memory: 64Mi
   limits:
-    cpu: 2
-    memory: 512Mi
+    cpu: 100m
```

Comment:
Do we have some basis for these limits?

Reply:
it's based on the data from [this graph](https://app.datadoghq.com/dashboard/svn-yjm-8zc/cf--grafbase-prod?fromUser=true&refresh_mode=sliding&tpl_var_deployment%5B0%5D=gateway&from_ts=1740395700348&to_ts=1741000500348&live=true)